src/sci/impl/protocols.cljc

```diff
@@ -241,14 +245,7 @@
         (or #?(:clj (contains? sats (symbol "class java.lang.Object"))
                :cljs (contains? sats (symbol extend-default-val)))
             (when-let [t (types/type-impl obj)]
-              #_{:clj-kondo/ignore [:redundant-let]}
-              (let [t (cond
-                        #?(:clj (class? t))
-                        #?(:clj (symbol (.getName ^Class t)))
-                        (instance? sci.lang.Type t)
-                        (symbol (str t))
-                        :else t)]
-                (contains? sats t)))))
+              (contains? sats (symbol (type->str t))))))
```

Comment:
We could maybe just keep track as a set of strings without converting to symbol

Reply:
I'll make that change 👍🏻

Reply:
OK, I'll await your change, or if you're busy, I can also finish this.